Darktrace event collector (#27542)
demisto/content: Packs/Darktrace/Integrations/DarktraceEventCollector/DarktraceEventCollector_description.md

## Darktrace

> To configure the connection to your Darktrace instance, you will provide:

Reviewer: ```suggestion
To configure the connection to your Darktrace instance, provide the following:
```
